physical/postgresql.go

```diff
@@ -69,8 +69,8 @@ func newPostgreSQLBackend(conf map[string]string) (Backend, error) {
 		"put":    put_statement,
 		"get":    "SELECT value FROM " + quoted_table + " WHERE path = $1 AND key = $2",
 		"delete": "DELETE FROM " + quoted_table + " WHERE path = $1 AND key = $2",
-		"list":   "SELECT key FROM " + quoted_table + " WHERE path = $1" +
-				"UNION SELECT substr(path, length($1)+1) FROM " + quoted_table + "WHERE parent_path = $1",
+		"list": "SELECT key FROM " + quoted_table + " WHERE path = $1" +
```

Comment:
I'm a stickler for putting these into different commits, but not necessarily PRs.  Good to know, however.